crates/ra_hir/src/source_binder.rs

```diff
@@ -17,17 +31,6 @@ use ra_syntax::{
 };
 use rustc_hash::{FxHashMap, FxHashSet};
 
```

Comment:
Let's keep the current ordering of imports:

```
std
// blank line
other::crate
// blank line
crate::
```

Reply:
Order is changed.